src/sample/courseRegController.java

```diff
@@ -157,10 +268,8 @@ public void addcourse() {
       // Clear the searchCourse Textfield after a course is registered.
       searchCourse.clear();
 
-      // Clear the Course Details after a course is registered.
-      courseTitle.setText("");
-      time.setText("");
-      desc.setText("");
+      addCourseButton.setDisable(true);
+      removeCourseButton.setDisable(true);
```

Comment:
why disable the buttons after adding courses? next time someone want to add then cannot add course ady